update container-service-tutorial-kubernetes-deploy-cluster.md

lavanyasubbarayalu opened the pull request:
az acs kubernetes get-credentials --resource-group myResourceGroup --name myK8SCluster command, gives out authentication failed error. If previous commands on creating kubernetes cluster gives some errors, then ssh key is not getting properly configured, giving out authentication failed errors, when connecting to the cluster. 
Scenario I have received authentication issue : Tried creating the kubernetes cluster with the az acs create --orchestrator-type kubernetes command and received errors that deployment is failed while provisioning cluster in master VM. Please see the error screenshot.

![acs k8scluster deployment failed](https://user-images.githubusercontent.com/20502019/31585954-e68c4182-b1e7-11e7-9392-8353d1b7e92a.png)

I have created new kubernetes cluster through portal with ssh key set through the portal to the master VM. Once az acs get-credentials command is executed, I received the authentication failed errors,  and fixed the same, by resetting and adding the ssh key to the ssh agent in the machine, where you are trying to connect to the cluster.

Provided troubleshooting notes, to fix the issue and proceed with next command mentioned in the tutorial.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

lavanyasubbarayalu committed:
update container-service-tutorial-kubernetes-deploy-cluster.md

az acs kubernetes get-credentials --resource-group myResourceGroup --name myK8SCluster command, gives out authentication failed error. If previous commands on creating kubernetes cluster gives some errors, then ssh key is not getting properly configured, giving out authentication failed errors, when connecting to the cluster. 
Provided troubleshooting notes, to fix the issue and proceed with next command.
